src/main/java/com/iridium/iridiumskyblock/gui/BlockValueGUI.java

```diff
@@ -73,7 +82,15 @@ public void addContent(Inventory inventory) {
      */
     @Override
     public void onInventoryClick(InventoryClickEvent event) {
-        // Don't do anything here, it gets cancelled automatically
+        Player player = (Player) event.getWhoClicked();
+        if (event.getSlot() == getNoItemGUI().size - 7 && page > 1) {
+            player.openInventory(new BlockValueGUI(page - 1, guiType).getInventory());
```

Comment:
In other GUIs we just changed the value of the page variable and called `player.openInventory(getInventory())`

Reply:
i did 👍🏻

Reply:
1) Do `page++` instead of `page+=1` (the indentation is bad too)
2) Remove the page argument from the constructor

Reply:
off why i did like this idk lol